util/test/perf/perfgraph.js

```diff
@@ -187,12 +187,21 @@ function getNextDivs(afterDiv, afterLDiv) {
   screenshotToggle.style.visibility = 'hidden';
   gspacer.appendChild(screenshotToggle);
 
+  // create a close graph button and put it next to the screnshot button
+  var closeGraphToggle = document.createElement('input');
+  closeGraphToggle.type = 'button';
+  closeGraphToggle.className = 'toggle';
+  closeGraphToggle.value = 'close';
+  closeGraphToggle.style.visibility = 'hidden';
+  gspacer.appendChild(closeGraphToggle);
+
   return {
     div: div,
```

Comment:
Nope, looks like a mistake added in 42b30afb when I went from 4 spaces per indent down to 2